ui/analyse/src/explorer/explorerConfig.ts

```diff
@@ -349,10 +350,7 @@ const playerModal = (ctrl: ExplorerConfigCtrl) => {
             spellcheck: 'false',
           },
           hook: onInsert<HTMLInputElement>(input =>
-            site.asset
-              .userComplete({ input, tag: 'span', onSelect: v => onSelect(v.name) })
-              .then(() => input.focus()),
-          ),
+            userComplete({ input, focus: true, tag: 'span', onSelect: v => onSelect(v.name) })),
```

Comment:
we lost `input.focus()` here. I tried it and indeed the input is no longer focused.